sources/platform/integrations/telegram.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Telegram integration through Zapier
+sidebar_label: Telegram
+description: Learn how to integrate your Apify Actors with Zapier.
+sidebar_position: 11.13
+slug: /integrations/telegram
+---
+
+**Learn how to integrate your Apify Actors with Telegram through Zapier.**
+
+---
+
+With [Apify integration for Zapier](https://zapier.com/apps/apify/integrations), you can connect your Apify Actors to Slack, Trello, Google Sheets, Dropbox, Salesforce, and loads more.
+
+Your Zapier workflows can start Apify Actors or tasks, fetch items from a dataset, set and get records from key-value stores, or find Actor or task runs.
+
+You can use the Zapier integration to trigger a workflow whenever an Actor or a task finishes.
+
+Completementary to the following guide we've created a detailed video, that will guide you through whole process of setting up your Telegram integratin through Zapier.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/XldEuQleq3c?si=86qbdrzWpVLoY_fr" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Connect Apify with Zapier
+
+To use the Apify integration on Zapier, you will need to:
+
+- Have an [Apify account](https://console.apify.com/).
+- Have a [Zapier account](https://zapier.com/).
+
+### Step 1: Create Zap and find Apify on Zapier
+
+Once you have your Zapier account ready and you are successfully logged in, you can create your first Zap.
+
+Go to the Zaps section and find the **Create Zap** button.
+
+In the create Zap form, you can choose whether you want to use Apify as the trigger or action for the Zap.
+
+Click on Trigger and find Apify using the search box.
+
+![Zapier choosing source of trigger](./images/zapier-trigger.png)
+
+You have two possible triggers that you can choose while setting up your Telegram integration
+
+- Finished Actor Run - triggers upon the completion of a selected Actor run.
+
+- Finished Task Run - triggers upon the completion of a selected Actor task run.
+
+![Zapier choosing trigger event](./images/zapier-zap-choose-event.png)
+
+:::info Available Actors & Tasks
+
+Please note that only Actor runs or Actor task runs that you previously executed will be available to choose from.
```

Comment:
Itsn't this

```suggestion
Please note that only Actors or Actor tasks that you previously run will be available to choose from.
```

? You are choosing an Actor or Task not exact run, right? But in fact, is this true? I think that you can now select any Actor or Task without running it previously.

Reply:
Yup, good catch it should have been Actor and Actor tasks not Actor run and Actor task runs.

Regarding the second point, has this changed within the last month since the publication of the video guide? I took the info from the video, and within it we have disclaimer that only Actors that were previously run will show up within Zapier when using Apify as trigger.

Reply:
I removed integration from title in each case except Telegram, since this one is through 3rd party. I think this approach works best as each doc is already under `Integrations` folder in the sidebar.

Reply:
@drobnikj would know when this was changed - you updated the Zapier integration in autumn to be able to select any Actor, right? So it might be incorrect in the video which is not a big deal IMHO.

Reply:
I spoke with @drobnikj yesterday, when using Apify as trigger you can only use previously run Actors or Actor tasks. So the information is correct both here and within the video.